server/resource/v1/kv_resource.go

```diff
@@ -244,6 +261,40 @@ func (r *KVResource) URLPatterns() []restful.Route {
 			},
 			Consumes: []string{"application/json"},
 			Produces: []string{"application/json"},
+		}, {
+			Method:           http.MethodDelete,
+			Path:             "/v1/kv/{ids}",
+			ResourceFuncName: "Delete",
+			FuncDesc:         "delete key by id,seperated by ','",
+			Parameters: []*restful.Parameters{
+				{
+					DataType:  "string",
+					Name:      "X-Domain-Name",
+					ParamType: goRestful.HeaderParameterKind,
+				}, {
+					DataType:  "string",
+					Name:      "ids",
+					ParamType: goRestful.PathParameterKind,
+					Desc: "The id strings to be removed are separated by ',',If the actual number of deletions " +
+						"and the number of parameters are not equal, no error will be returned and only warn log will be printed.",
+				},
+			},
+			Returns: []*restful.Returns{
+				{
+					Code:    http.StatusNoContent,
+					Message: "Delete success",
+				},
+				{
+					Code:    http.StatusBadRequest,
+					Message: "Failed,check url",
+				},
+				{
+					Code:    http.StatusInternalServerError,
+					Message: "Server error",
+				},
+			},
+			Consumes: []string{goRestful.MIME_JSON},
```

Comment:
shouldn't add consume and produce, it is a simple delete method

Reply:
done